tests/cli/test_cmds_spark_run.py

```diff
@@ -1450,3 +1450,75 @@ def test_get_aws_credentials():
         RoleSessionName=mock.ANY,
         WebIdentityToken="token-content",
     )
+
+
+@mock.patch("service_configuration_lib.spark_config.use_aws_profile", autospec=False)
+@mock.patch("service_configuration_lib.spark_config.Session", autospec=True)
+def test_get_aws_credentials_session(mock_boto3_session, mock_use_aws_profile):
+    # prioritize session over `profile_name` if both are provided
+    from service_configuration_lib.spark_config import get_aws_credentials
```

Comment:
Any reason to not have imports at the top? LGTM otherwise

Reply:
Moved all the import statements!